static/bidder-info/tpmn.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: "https://gat.tpmn.io/ortb/pbs_bidder"
+maintainer:
+  email: "prebid@tpmn.io"
+modifyingVastXmlAllowed: true
+endpointCompression: GZIP
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
```

Comment:
json tests doesn't cover all these paths. please add missing tests for all supported capabilities

Reply:
update commit :

[1dce2d4](https://github.com/prebid/prebid-server/pull/2947/commits/1dce2d4789e2cd61f91fb3ef5aa606d2713c5289)

[15dc048](https://github.com/prebid/prebid-server/pull/2947/commits/15dc0487250ed476324b4512938bdea052cfb3d8)